src/main/java/uk/gov/pay/directdebit/mandate/params/MandateSearchParams.java

```diff
@@ -107,7 +110,13 @@ public Integer getPage() {
     public Integer getDisplaySize() {
         return displaySize;
     }
-    
+
+    public List<MandateState> getInternalStates() {
+        return Arrays.stream(MandateState.values())
+                .filter(state -> state.toExternal().getState().equalsIgnoreCase(this.externalMandateState))
```

Comment:
Not 100% sure we need this to be case-insensitive; it doesn’t look like it is for card connector

Reply:
Done!